app/migrations/Version20151231002659.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Sylius\Migrations;
+
+use Doctrine\DBAL\Migrations\AbstractMigration;
+use Doctrine\DBAL\Schema\Schema;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+class Version20151231002659 extends AbstractMigration
+{
+    /**
+     * @param Schema $schema
+     */
+    public function up(Schema $schema)
+    {
+        // this up() migration is auto-generated, please modify it to your needs
+        $this->abortIf($this->connection->getDatabasePlatform()->getName() != 'mysql', 'Migration can only be executed safely on \'mysql\'.');
+
+        $this->addSql('ALTER TABLE sylius_order_item ADD units_total INT NOT NULL');
```

Comment:
Such migrations are often failing on database that contain data, this would fail that this field cannot be `null` but it will be, as every new field in existing data is set to `null`.

First this should contain `null` as default value, later there should be `update` sql that sets 0 (or other value) and after that there should be set `not null` on this new field.